docs/root/operations/admin.rst

```diff
@@ -500,6 +514,144 @@ modify different aspects of the server:
   Outputs statistics that Envoy has updated (counters incremented at least once,
   gauges changed at least once, and histograms added to at least once) in JSON format.
 
+  .. http:get:: /stats?format=json&histogram_buckets=cumulative
+
+  Changes histogram output to display cumulative buckets with upper bounds.
+  All values below the upper bound are included even if they are placed into other buckets.
+  Compatible with ``usedonly`` and ``filter``.
+
+  Example histogram output:
+
+  .. code-block:: json
+
+    {
+      "histograms": [
+        {
+          "name": "example_histogram",
+          "supported_buckets": [
+            0.5, 1, 5, 10, 25, 50, 100, 250, 500, 1000, 2500, 5000, 10000, 30000, 60000, 300000, 600000, 1800000, 3600000
+          ],
+          "computed_buckets": [
+            {"interval": 0, "cumulative": 0},
```

Comment:
WDYT of adding a comments saying that these lines are ordered based on "supported_buckets", maybe above the json sample.

Reply:
Should the arrays be combined, so that each element of "computed_buckets" (which would probably be renamed to "buckets" is `{"upper_bound"=10, "interval"=0, "cumulative"=0}`? Is there any value to splitting them into two parallel/correlated arrays?

Reply:
Yeah I agree with that. I thought the reason to make them parallel is to factor out the common structure for all histograms and reduce output size, but now I'm not sure if that's actually happening in this case.